### PR TITLE
Maestro bootstrap shell script & `docs/MAESTRO.md`

### PR DESCRIPTION
**Pull request by itayplav**
### Dependencies

none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204357017841273

### CODE REVIEW

File: docs/MAESTRO.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+# /Library/Developer/CommandLineTools
+
+```
+
+4. Next, run the convenience shell script to install all dependencies required for Maestro, including Maestro itself:
+
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/EdgeApp/edge-react-gui/maestro.sh)"
```
Comment (itayplav) [inline docs/MAESTRO.md:72]: Note for reviewers: this command will not work until this PR has been merged - until then the correct command is:

```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/EdgeApp/edge-react-gui/itay/maestro/readme/maestro.sh)"
```

**Review by paullinator — CHANGES_REQUESTED** [review]

**Review by paullinator — APPROVED** [review]
Approved with my fixup commit.

File: maestro.sh [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    echo "$path_to_add already exists in PATH"
+  fi
+}
+
+NODE_VERSION="16.20.0"
+JAVA_VERSION="11.0.19-zulu"
+
+JAVA_SDKMAN_URL="https://get.sdkman.io"
+MAESTRO_INSTALL_URL="https://get.maestro.mobile.dev"
+EDGE_REACT_GUI_REPO_URL="https://github.com/EdgeApp/edge-react-gui"
```
[the remark this answers is not in the document]
Comment (itayplav) [reply maestro.sh]: I think your saying new repo ?

File: maestro.sh [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+      echo "Failed to clone edge-react-gui repository."
+      return 1
+    fi
+  fi
+}
+
+start
+
+echo 'Installation of Maestro, its dependencies, and edge-react-gui completed.'
+exec $SHELL -l
```
[the remark this answers is not in the document]
Comment (itayplav) [reply maestro.sh]: vestigial sanity check to have an actual brand new shell..woops!

**Pull request merged**